src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php

```diff
@@ -183,6 +184,7 @@ private function fixPhpUnitClass(Tokens $tokens, $startIndex, $endIndex)
         $tokensAnalyzer = new TokensAnalyzer($tokens);
 
         for ($i = $endIndex - 1; $i > $startIndex; --$i) {
+            // @noinspection NotOptimalIfConditionsInspection
```

Comment:
:-1: for adding tools instructions in the code.

Reply:
Right, this does not look good for the code, removed.